radio/src/main.cpp

```diff
@@ -196,29 +195,6 @@ void handleUsbConnection()
 #endif // defined(STM32) && !defined(SIMU)
 }
 
-#if defined(JACK_DETECT_GPIO) && !defined(SIMU)
-bool isJackPlugged()
-{
-  // debounce
-  static bool debounced_state = 0;
-  static bool last_state = 0;
-
-  if (GPIO_ReadInputDataBit(JACK_DETECT_GPIO, JACK_DETECT_GPIO_PIN)) {
-    if (!last_state) {
-      debounced_state = false;
-    }
-    last_state = false;
-  }
-  else {
-    if (last_state) {
-      debounced_state = true;
-    }
-    last_state = true;
-  }
-  return debounced_state;
-}
-#endif
-
```

Comment:
Was about to rebase this rather than bother you since it looks straightforward... but, um... did you remove this by accident? As `isJackPlugged()` is still used in code...